database/postgres/postgres.go

```diff
@@ -266,13 +277,23 @@ func runesLastIndex(input []rune, target rune) int {
 	return -1
 }
 
+func (p *Postgres) quoteIdentifier(name string) string {
+	if p.config.MigrationsTableHasSchema {
+		if strings.Index(name, ".") != -1 {
+			splited := strings.Split(name, ".")
```

Comment:
We need to handle the case when there are multiple `.`
I think it's safer to error if there is more than a single split (2 elements). This will make this method more clunky to use since all callers will need to check for the error...
An alternative is to use `SplitN`, but I dislike this approach since it could cause issues for users due to the unexpected behavior. I guess if we document this it's fine...